dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -291,6 +297,7 @@
         "KeyPhraseExtraction",
         "EntityLinking",
         "Healthcare",
+        "CustomHealthcare",
```

Comment:
I think we're missing the corresponding CustomHealthcareLROTask object that sets the discriminator. For example, the one for HealthcareLROTask: https://github.com/Azure/azure-rest-api-specs/blob/7eda52e0792c584c722016ef5ee533f923ba41cf/dev/cognitiveservices/data-plane/Language/analyzetext.json#L856-L869

Reply:
Added. Thanks!